QA.md

```diff
@@ -0,0 +1,89 @@
+<!--
+QA.md
+Copyright 2016 Canonical Ltd.
+This work is licensed under the Creative Commons Attribution-Share Alike 3.0
+Unported License. To view a copy of this license, visit
+http://creativecommons.org/licenses/by-sa/3.0/ or send a letter to Creative
+Commons, 171 Second Street, Suite 300, San Francisco, California, 94105, USA.
+-->
+
+# Juju GUI QA Process #
+
+Before releasing the Juju GUI charm it needs to be thoroughly tested across a
+large number of scenarios. This document is not exhaustive but lists the best
+practice checklist for what should be an acceptable QA.
+
+Given the large number of scenarios, expect this to take at least four
+hours. Currently it is a manual process but we hope to partially automate it
+soon.
+
+## Various ways to launch the GUI ##
+
+* Sandbox mode
+  - In the juju-gui code, `make qa-server`
+
+* Deployed via the charm using juju v1.24
+  - Test single model only.
+  ```
+  juju bootstrap
+  make deploy
+  ```
+
+* Deployed via the charm using juju v1.25
+
+  Model only with controller.
+    * No access to admin environment
+    * Cannot create new model
+  ```
+  export JUJU_DEV_FEATURE_FLAGS=jes
```

Comment:
as above - to make this a little more future proof maybe some instructions on how to get 1.25